command/src/main/scala/nl.knaw.dans.easy.stage.command/Command.scala

```diff
@@ -34,7 +36,7 @@ object Command extends App {
   implicit val settings: Settings = new Settings(
     ownerId = getUserId(clo.deposit()),
     submissionTimestamp = if (clo.submissionTimestamp.isSupplied) clo.submissionTimestamp()
-                          else DateTime.now,
+    else DateTime.now,
```

Comment:
if this formatting change was done by IntelliJ, it means you don't have the right formatter installed 😉
See https://github.com/DANS-KNAW/easy-module-archetype/blob/master/dans-intellij-codestyles.xml and https://github.com/DANS-KNAW/easy-module-archetype/blob/master/dans-intellij-inspections.xml

Reply:
Same regarding formatting in other places/files.